### PR TITLE
Feature/filter&cards layout

### PR DESCRIPTION
**Pull request by beliavskaja**
- made the adaptive layout of cards and buttons
- added a popup for adding a filter later
- changed the size of cards
- changed buttons in the card

Screenshot 2021-11-03 at 18 12 51
Screenshot 2021-11-03 at 18 13 06
Screenshot 2021-11-04 at 10 28 58

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { Switch, Route, Redirect } from 'react-router-dom';
 import { connect } from 'react-redux';
 import { ThemeProvider } from '@mui/material/styles';
 import { theme } from './utils';
-import { Home, Login } from './pages';
+import { Home, Login, Internships } from './pages';
```
Comment (svtslv17) [inline src/App.jsx:7]:
```suggestion
import { Home, Login } from './pages';
```

**Comment by itsnotpolza** [comment]
Please make action buttons on mobile responsive as well (moving buttons under the title on mobile will be fine)

File: src/components/internshipCard/InternshipCard.jsx [hunk 2]
```diff
@@ -15,40 +14,35 @@ import { getFieldLabel } from '../../utils';
 export const InternshipCard = ({ data }) => {
   const { imageUrl, title, dateInterval, numberOfMembers } = data;
```
Comment (itsnotpolza) [inline src/components/internshipCard/InternshipCard.jsx:15]:
```suggestion
export const InternshipCard = ({ data: { imageUrl, title, dateInterval, numberOfMembers} }) => {
```

File: src/pages/internships/Internships.jsx [hunk 3]
```diff
@@ -2,56 +2,65 @@ import React from 'react';
...
+            flexDirection="row"
+            justifyContent="space-between"
+            marginRight="100px"
+          >
+            <IconButton>
+              <ManageSearchIcon fontSize="large" />
+            </IconButton>
+            <InternshipFilter />
+            <Button variant="outlined" size="small">
+              {getFieldLabel('internships.btn.add.program')}
```
Comment (itsnotpolza) [inline src/pages/internships/Internships.jsx:51]: don't shorten button to btn, full words are okay

**Review by nsukhotskaya — APPROVED** [review]

**Review by OstroumovAlex — APPROVED** [review]

**Pull request merged**